setup.sh

```diff
@@ -165,9 +197,9 @@ datasetId=`echo ${dataSet} | cut -d'"' -f2 | cut -d'/' -f3`
 echo "Enter the number of Experience events to publish"
 read count
 
-echo "org=${IMS_ORG}" > ./application.conf
-echo "schema=${schema}" >> ./application.conf
-echo "dataset=${datasetId}" >> ./application.conf
-echo "topic=${connectTopicName}" >> ./application.conf
+echo "org=${IMS_ORG}" > ${PWD}/application.conf
+echo "schema=${schema}" >> ${PWD}/application.conf
+echo "dataset=${datasetId}" >> ${PWD}/application.conf
+echo "topic=${connectTopicName}" >> ${PWD}/application.conf
```

Comment:
I did not get a application.conf created with the values after running thru the test

Reply:
This was a user issue, pls ignore.